### PR TITLE
Meta tags and basic SEO

### PR DESCRIPTION
**Pull request by Alph3ga**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,9 +2,22 @@
...
+    <meta name="theme-color" content="#04142E"> <!--PLACEHOLDER, MIGHT CHANGE WHEN WE DECIDE THEME-->
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Kharagpur Winter of Code | For the love of open source</title>
+    <!--Social Media card tags-->
+    <meta property="og:title" content="Kharagpur Winter of Code">
+    <meta property="og:type" content="website" />
+    <meta property="og:image" content="/kwoc_logo.png"> 
+    <meta property="og:url" content="https://kwoc.kossiitkgp.org/">
+    <meta name="twitter:card" content="summary_large_image">
+    <meta property="og:description" content="Oppurtunity for students to explore the world of Free and Open Source Software.">
```
Comment (harshkhandeparkar) [inline index.html]:
```suggestion
    <meta property="og:description" content="Opportunity for students to explore the world of Free and Open Source Software.">
```

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

**Comment by rajivharlalka** [comment]
LGTM, can merge this! @Alph3ga

**Review by rajivharlalka — DISMISSED** [review]

**Comment by rajivharlalka** [comment]
@harshkhandeparkar approve krde?

**Comment by harshkhandeparkar** [comment]
> @harshkhandeparkar approve krde?

There is one issue that still needs to be resolved. Check Slack.

**Review by harshkhandeparkar — APPROVED** [review]
Wasted enough time to make it work on Whatsapp. No more time needs to be wasted.

**Pull request merged**